### ISSUE TITLE
One build for e2e and component tests

### ISSUE THREAD

**Issue by aleksey-ilin** [cypress-io/cypress#27832]
### What would you like?

I have e2e and component tests in my projects. I want to build project 1 time and use it for e2e and component tests. Is it available?

### Why is this needed?

It will reduce test time.

**Comment by aleksey-ilin** [comment]
Hello, @nagash77 . Can you answer to my question?

**Comment by jennifer-shehane** [comment]
@aleksey-ilin No, this is not available. It is quite complicated to implement and we discussed this option a lot when introducing component testing. 

I’ll leave this open as a feature request.
